.travis.yml

```diff
@@ -3,10 +3,14 @@ sudo: false
 language: node_js
 node_js:
   - "7"
+notifications:
+  email: false
```

Comment:
This is to disable Travis email notification, apparently you have to manually put this in the travis.yml file